tests/src/test/java/alluxio/server/ft/journal/TableMasterJournalIntegrationTest.java

```diff
@@ -92,8 +96,12 @@ public void journalAttachDb() throws Exception {
     TableMaster tableMaster =
         mCluster.getLocalAlluxioMaster().getMasterProcess().getMaster(TableMaster.class);
 
+    mException.expect(IOException.class);
+    tableMaster.getDatabase(DB_NAME);
```

Comment:
won't this stop the rest of the test from being run?

Reply:
doh.. why aren't we on Junit 5 yet?  I totally misunderstood the semantics of `expect`